osu.Game/Overlays/Mods/ModSelectOverlay.cs

```diff
@@ -186,18 +214,10 @@ private void load(OsuGameBase game, OsuColour colours, AudioManager audio)
 
             if (ShowTotalMultiplier)
             {
-                MainAreaContent.Add(new Container
+                aboveColumnsContent.Add(multiplierDisplay = new DifficultyMultiplierDisplay
                 {
-                    Anchor = Anchor.TopRight,
-                    Origin = Anchor.TopRight,
-                    AutoSizeAxes = Axes.X,
-                    Height = ModsEffectDisplay.HEIGHT,
-                    Margin = new MarginPadding { Horizontal = 100 },
-                    Child = multiplierDisplay = new DifficultyMultiplierDisplay
-                    {
-                        Anchor = Anchor.Centre,
-                        Origin = Anchor.Centre
-                    },
+                    Anchor = Anchor.Centre,
+                    Origin = Anchor.Centre
```

Comment:
why put `DifficultyMultiplierDisplay` at Centre, its original position was TopRight.
is there any reason

Reply:
There is no particular reason, it happened on occasion [here ](https://github.com/ppy/osu/pull/23414/commits/a6ca0497399d5e83faca452782c49fa74c2c406a) so I'll change it back once we establish search bar location.

Reply:
Search bar location is top left according to the Figma design
For reference:
![image](https://github.com/ppy/osu/assets/39100084/6fd35e86-aca1-4f35-a24a-57014fe27a21)

Reply:
@ItsShamed can you attach Figma link as well?

Reply:
https://www.figma.com/file/NkdREXr6wFkA99oLw8PnBv/Client%2FMod-Selection?type=design&node-id=2292%3A373&t=OtVVcubZ6HnO0Mur-1

Reply:
Also, I think that the search text font is too big, it should be the same size as the text for the difficulty multiplier imo. For reference, this is the text for the `ModsEffectDisplay`

https://github.com/ppy/osu/blob/7bc8908ca9c026fed1d831eb6e58df7624a8d614/osu.Game/Overlays/Mods/ModsEffectDisplay.cs#L106-L114

But since `TextBox` forces the text to be a certain font size using `CalculatedTextSize` I don't know how you can work around that.

Reply:
I've taken a look into font changing, and it seems to be a hard one. I don't think changing font is reasonable here, also other search boxes seems to have the same issue (see pic below)

![image](https://github.com/ppy/osu/assets/50776304/a1608ef3-51b7-4ed6-a5b6-7063a782e681)

Reply:
Why is this conversation resolved? Please put back the multiplier where it used to be for now.

Reply:
It's not resolved yet

Reply:
It was resolved when I found it. I unresolved it.

Reply:
Idk. Mb I missclicked :/